pkg/agent/handler/accessrequest.go

```diff
@@ -28,22 +29,27 @@ type arProvisioner interface {
 }
 
 type accessRequestHandler struct {
-	prov   arProvisioner
-	cache  agentcache.Manager
-	client client
+	prov      arProvisioner
+	cache     agentcache.Manager
+	agentType config.AgentType
+	client
```

Comment:
client client ?

Reply:
Go's syntactic sugar, however will update to avoid confusion